app/assets/stylesheets/footer.sass.scss

```diff
@@ -1,20 +1,24 @@
+footer {
+  &:before {
+    content: "";
+    width: 100%;
+    height: 20px;
+    background: transparent url("footer-bg.png") top center no-repeat;
+    position: absolute;
+    top: -16px;
+    left: 0;
+  }
+
+  position: relative;
+  font-size: 0.8em;
+  background-color: $footer-bg;
+  padding: 20px;
```

Comment:
Is it worth switching to `em` here (and in `generic.sass.scss`) rather than `px` as part of these changes?

Reply:
I'd rather not start yak shaving everything (there's a lot of yak to shave here). The main point here is for the same styling to apply on desktop and mobile, which I think this does without anything looking broken.